blaze/expr/collections.py

```diff
@@ -424,10 +424,35 @@ def join(lhs, rhs, on_left=None, on_right=None, how='inner'):
 
 join.__doc__ = Join.__doc__
 
+class IsIn(Expr):
```

Comment:
I have something similiar [here](https://github.com/cowlicks/blaze/commit/61979fb40486ef78f35ede56b84b4ce09f1654a3#diff-c9f69944277033e73dc99af44b3e7a86R493). But for some reason the breakpoint does not work when I try:
```python
blaze.Data(range(6)).isin([1, 2])
```